.swiftformat

```diff
@@ -91,6 +91,7 @@
     blockComments, \
     # docComments - Cannot enable due to our floating doc comment style
     isEmpty, \ # Rare false positives in situations where a type provides a `count` but not a `isEmpty` property.
+    markTypes, \
```

Comment:
I don't consider this rule beneficial. Xcode would then show the name of the type twice, right? 
<img width="171" alt="Screenshot 2024-10-21 at 7 22 42 PM" src="https://github.com/user-attachments/assets/a8ef8a43-7fe8-43b9-9cfd-ee7147e6addf">